libs/sdk-ui-dashboard/src/presentation/widget/insightMenu/DefaultDashboardInsightMenu/getDefaultInsightMenuItems.tsx

```diff
@@ -17,56 +19,167 @@ export type SchedulingDisabledReason = "incompatibleWidget" | "oldWidget" | "dis
  */
 export type AlertingDisabledReason = "noDestinations" | "oldWidget" | "disabledOnInsight";
 
+const getExportMenuItems = (
+    intl: IntlShape,
+    config: IUseInsightMenuConfig,
+    execution?: IExecutionResultEnvelope,
+    settings?: ISettings,
+): IInsightMenuItem[] => {
+    const {
+        isExportVisible,
+        isExportRawVisible,
+        exportPdfPresentationDisabled,
+        exportPowerPointPresentationDisabled,
+        exportXLSXDisabled,
+        exportCSVDisabled,
+        exportCSVRawDisabled,
+        onExportPdfPresentation,
+        onExportPowerPointPresentation,
+        onExportXLSX,
+        onExportCSV,
+        onExportRawCSV,
+    } = config;
+    const tooltip = getExportTooltip(execution, settings?.enableRawExports);
+    const presentationTooltip = intl.formatMessage({
+        id: "options.menu.export.presentation.unsupported.oldWidget",
+    });
+
+    return [
+        // Presentation exports section - only shown if isExportVisible is true
+        ...(isExportVisible
+            ? [
+                  {
+                      type: "button" as const,
+                      itemId: "ExportPdfPresentation",
+                      itemName: intl.formatMessage({ id: "options.menu.export.presentation.PDF" }),
+                      icon: "gd-icon-type-pdf",
+                      className: "gd-export-options-pdf-presentation",
+                      disabled: exportPdfPresentationDisabled,
+                      tooltip: exportPdfPresentationDisabled ? presentationTooltip : undefined,
+                      onClick: () => {
```

Comment:
I think all these handlers can be just inlined? `onClick: onExportPdfPresentation`

Reply:
fixed